.github/workflows/kvrocks.yaml

```diff
@@ -145,7 +145,11 @@ jobs:
       - name: Run Redis Tcl Test
         run: |
           cp $HOME/local/bin/redis-cli tests/tcl/redis-cli
-          cd tests/tcl && ./runtest
+          cd tests/tcl
+          ./runtest --dont-clean
+          SANITIZER_OUTPUT=$(grep "Sanitizer:" tests/tmp -r)
+          echo $SANITIZER_OUTPUT
+          if [[ $SANITIZER_OUTPUT ]]; then exit 1; fi
```

Comment:
I'd prefer to echo output in the `if` block.

Reply:
like `if [[ $SANITIZER_OUTPUT ]]; then echo $SANITIZER_OUTPUT; exit 1; fi` ?

Reply:
done